main/src/library/Fixpoint/Interpreter.flix

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2021 Benjamin Dahse
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use Fixpoint/Ram.{RamStmt, RelOp, RelSym, RamTerm, BoolExp, RowVar, toDenotation};
+use Fixpoint/Ast.Denotation;
+
+instance Order[Array[v]] with Order[v] {
```

Comment:
We should discuss this- because you are defining a *global* Order instance on arrays. IF this is a good idea then- at minimum- it should be moved to `Array.flix`.

Reply:
Moved to Array.flix for now.